akka-docs/src/main/paradox/stream/operators/Source-or-Flow/flatMapMerge.md

```diff
@@ -4,26 +4,39 @@ Transform each input element into a `Source` whose elements are then flattened i
 
 @ref[Nesting and flattening operators](../index.md#nesting-and-flattening-operators)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Flow.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala) { #flatMapMerge }
-
-@@@
+@apidoc[Flow.flatMapMerge](Flow) { scala="#flatMapMerge[T,M](breadth:Int,f:Out=%3Eakka.stream.Graph[akka.stream.SourceShape[T],M]):FlowOps.this.Repr[T]" java="#flatMapMerge(int,akka.japi.function.Function)" } 
 
 ## Description
 
 Transform each input element into a `Source` whose elements are then flattened into the output stream through
-merging. The maximum number of merged sources has to be specified.
+merging. The maximum number of merged sources has to be specified. When this is met `flatMapMerge` does not
+request any more elements meaning that it back pressures until one of the existing `Source`s completes. 
```

Comment:
Something about what order (no guaranteed order) elements are merged into the downstream perhaps?